lib/jekyll-import/importers/dotclear.rb

```diff
@@ -1,125 +1,182 @@
 # frozen_string_literal: true
 
-# Tested with dotClear 2.1.5
 module JekyllImport
   module Importers
     class Dotclear < Importer
       class << self
         def specify_options(c)
-          c.option "datafile", "--datafile PATH", "dotClear export file"
-          c.option "mediafolder", "--mediafolder PATH", "dotClear media export folder (media.zip inflated)"
+          c.option "datafile",    "--datafile PATH",   "Dotclear export file."
+          c.option "mediafolder", "--mediafolder DIR", "Dotclear media export folder (unpacked media.zip)."
         end
 
         def require_deps
-          JekyllImport.require_with_fallback(%w(
-            rubygems
-            fileutils
-            safe_yaml
-            date
-            active_support
-            active_support/core_ext/string/inflections
-            csv
-            pp
-          ))
+          JekyllImport.require_with_fallback(%w())
```

Comment:
Omit this if we don't require anything.

Reply:
This is required for compatibility with the local plugin `docs/_plugins/importer_metadata.rb`.